database/migrations/2017_12_04_113000_create_users_table.php

```diff
@@ -18,6 +18,7 @@ public function up()
             $table->string('name');
             $table->string('email')->unique();
             $table->string('password');
+            $table->string('role')->default('CC_User');
```

Comment:
Are we liking the Uppercase letters and the redundant 'user'? Or would it be better just cc and fm? Or centre & admin - what is fm?

Reply:
FM is Food Matters; CC is ChildrensCentre
You're right it's obscure.
FoodMattersUser and CentreUser (there may be a CentreAdmin in future)?

Reply:
But don't use cases either probably...